small analysis of result table (#74)
MrLongbottom/P10-Article: tables/metric_results.tex

		Taxonomy-Topic Model & - & 0.660 & 0.709 \\
	\end{tabular}
	\label{tab:metric_results}
\end{table}

> From \autoref{tab:metric_results}, we can see that the Author and Category-Topic models are performing the worst, whereas the Taxonomy-Topic model is out performing all other models.

Reviewer: ```suggestion
From \autoref{tab:metric_results}, we can see that the Author and Category-Topic models are performing the worst, whereas the Taxonomy-Topic model is outperforming all other models.
```

However, the run time of the Taxonomy model is worse than the standard \gls{lda}.

> For the \gls{lda} to compute 50 epochs, that takes around 6-8 hours, depending on the CPU. 

Reviewer: ```suggestion
It takes about 6-8 hours to compute 50 epochs for the \gls{lda} model, depending on the CPU. 
```

> The Taxonomy model running a 5 layered pachinko model, took 132 hours before completing the 50 epochs.

Reviewer: ```suggestion
The Taxonomy model running a 5 layered \gls{pam} took 132 hours before completing the 50 epochs.
```
